spreadsheets.js

```diff
@@ -506,16 +506,19 @@ function find_players(tokens){
 
     //assuming we found 2 tokens, we should convert them to player names
     if(player_tokens.length == 2){
-        players.white = player_tokens[0];
-        players.black = player_tokens[1];
+        //remove punctuation and store. this fixes losts of stuff.
+        //most frequent issue is the : added after a name by slack
+        players.white = player_tokens[0].replace(/[:,.-]/, "");
+        players.black = player_tokens[1].replace(/[:,.-]/, "");
     }
 
     return players;
 }
 
 function filter_player_tokens(tokens){
     return _.filter(tokens, function(token){
-        return /^\<@[A-Z0-9]+\>$/.test(token);
+        //matches slack uer ids: <@[A-Z0-9]>[:]*
+        return /^\<@[A-Z0-9]+\>[:]*$/.test(token);
```

Comment:
Yeah, let's make a ticket for getting these regexes unified so we don't have to keep writing them for each individual spot.